### ISSUE TITLE
Unable to have terrible humor with my bot 

### ISSUE THREAD

**Issue by stahnma** [stephenyeargin/hubot-dad-jokes#9]
```
[Wed Jan 03 2024 17:48:16 GMT+0000 (Coordinated Universal Time)] DEBUG Executing listener callback for Message '! dadjoke'
[Wed Jan 03 2024 17:48:16 GMT+0000 (Coordinated Universal Time)] ERROR SyntaxError: Unexpected token < in JSON at position 0
  at JSON.parse (<anonymous>:null:null)
  at /var/lib/goldiflox/node_modules/hubot-dad-jokes/src/dad-jokes.js:19:25
  at IncomingMessage.<anonymous> (/var/lib/goldiflox/node_modules/hubot/src/httpclient.js:133:38)
  at IncomingMessage.emit (node:events:525:35)
  at endReadableNT (node:internal/streams/readable:1359:12)
  at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

Version is 1.1.0.

**Comment by stephenyeargin** [comment]
Sigh.

```
x-amzn-waf-action: captcha
```

**Comment by stephenyeargin** [comment]
@stahnma One option would be to have a cache to fall back to when the initial request fails (would net ~49 jokes). Also should have added some better error handling.

**Comment by stephenyeargin** [comment]
Fixed in `v1.1.1` (in that it falls back to cached jokes -- good chance it never pulls live data again)
